tipg/sql/dbcatalog.sql

```diff
@@ -262,8 +263,8 @@ CREATE OR REPLACE FUNCTION pg_temp.tipg_catalog(
             AND '' != ANY(proargnames)
             AND has_function_privilege(oid, 'execute')
             AND provariadic=0
-            AND (functions IS NULL OR concat(p.pronamespace::regnamespace::text, '.', proname::text) = ANY (functions))
-            AND (exclude_functions IS NULL OR concat(p.pronamespace::regnamespace::text,'.',proname::text) != ANY (exclude_functions))
+            AND (exclude_functions IS NULL OR concat(p.pronamespace::regnamespace::text,'.',proname::text) <> ALL (exclude_functions))
+            AND (functions IS NULL OR concat(p.pronamespace::regnamespace::text,'.',proname::text) = ANY (functions))
```

Comment:
I don't understand but the include/exclude filter do not work for function (see the failing tests)
cc @bitner

Reply:
Ahh, this is because the actual namespace of the function isn't pg_temp -- pg_temp is an automatic alias for a temp schema that is created behind the scenes